.github/workflows/appimage.yml

```diff
@@ -0,0 +1,96 @@
+name: Build AppImage Packages
+
+on:
+  # Run on a schedule to get monthly updates
+  schedule:
+    - cron: "0 0 28 * *"

```

Comment:
I just want to make sure that this is being used as intended. The reason I added the `cron` statement to the Linux workflow was to make sure that the underlying containers get refreshed with updated deps each month so we don't have to do a security dance if a dep gets pwnd. This could be useful for making sure that the README has packages available with the 90-day default expiry time.